runtime/plugin/lsp.vim

```diff
@@ -0,0 +1,28 @@
+" TODO(tjdevries): Move this to autoload?
+
+execute('luafile ' . expand('<sfile>:h') . '/../lua/lsp/plugin.lua')
+
+function! LSP_Start() abort
+lua << EOF
+client_start()
+EOF
+endfunction
+
+" TODO: Automatically open file if not opened already
+function! LSP_References() abort
+lua << EOF
```

Comment:
`lua lsp.client_references()`, indented:

1. Do not use ugly `<<EOF` blocks.
2. All globals must be namespaced. This is not the only plugin using lua. I personally would prefer a single global per plugin or no globals at all (use `lua require("…").func()`).